### ISSUE TITLE
Old state events not rolling off the site

### ISSUE THREAD

**Issue by nathanmwilliams** [townhallproject/admin#62]
Seemingly tied to the fact they all have "Date Not Valid" flags.

(screenshot taken from Live on June 12)

![Cynthia old event](https://user-images.githubusercontent.com/25805849/59396677-b70bbf80-8d3e-11e9-90ab-99d42b05f723.png)

**Comment by TheMiloAnderson** [comment]
Hi Nathan, I gather that something is supposed to happen when the date has passed, but could you clarify what you mean by 'rolling off the site'?

**Comment by nathanmwilliams** [comment]
Yeah, so when events are in the past, they shouldn't be displaying on townhallproject.com. I've found tele-town hall events and some state events seem to stick around until I manually Archive them (which can be time consuming)
